docs/src/ci-intro.md

```diff
@@ -0,0 +1,552 @@
+---
+id: ci-intro
+title: "CI GitHub Actions"
+---
+
+## Introduction
+* langs: js
+
+Playwright tests can be executed on CI using GitHub actions. When installing Playwright you are given the option to add a [GitHub Actions](https://docs.github.com/en/actions). This creates a `playwright.yml` file inside a `.github/workflows` folder containing everything you need so that your tests run on each push and pull request into the main/master branch.+
+
+#### You will learn
+* langs: js
+
+- [How to run tests on push/pull_request](/ci-intro.md#on-pushpull_request)
+- [How to view test logs](/ci-intro.md#viewing-test-logs)
+- [How to view the HTML report](/ci-intro.md#viewing-the-html-report)
+- [How to view the trace](/ci-intro.md#viewing-the-trace)
+- [How to publish report on the web](/ci-intro.md#publishing-report-on-the-web)
+
+
+## Introduction
+* langs: python, java, csharp
+
+Playwright tests can be executed on CI using GitHub Actions. To add a [GitHub Actions](https://docs.github.com/en/actions) file first create `.github/workflows` folder and inside it add a `playwright.yml` file containing the example code below so that your tests will run on each push and pull request for the main/master branch.
+
+#### You will learn
+* langs: python, java, csharp
+  
+- [How to run tests on push/pull_request](/ci.md#on-pushpull_request)
+- [How to view test logs](/ci-intro.md#viewing-test-logs)
+- [How to view the trace](/ci-intro.md#viewing-the-trace)
+
+
+
+## Setting up GitHub Actions
+
+### On push/pull_request
+* langs: js
+
+Tests will run on push or pull request on branches main/master. The [workflow](https://docs.github.com/en/actions/using-workflows/about-workflows) will install all dependencies, install Playwright and then run the tests. It will also create the HTML report.
+
+```yml js
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
```

Comment:
nit: let's keep just `main` here and in the text?

Reply:
I'd keep both for now, since a lot of projects still rely on master. Otherwise we'd have to go ahead and change also create-playwright and the other places where we mention it.